build.gradle

```diff
@@ -80,7 +80,7 @@ allprojects {
 
     dependencyManagement {
         imports {
-            mavenBom("org.springframework.boot:spring-boot-dependencies:2.6.7")
+            mavenBom(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES)
```

Comment:
does this ensure that the version will be automatically upgraded?

Reply:
I think this was a mistake, just reverted the change